From 808271dd2df8542c47dd8b04b74d167ef8e9a019 Mon Sep 17 00:00:00 2001
From: Shi Wenjun <b42754@freescale.com>
Date: Fri, 14 Dec 2012 14:20:05 +0800
Subject: [PATCH 156/310] ENGR00236719 Can't enter deep Suspend due to 'RILJ'
 activated with EM770W.

Pick up ril property settings from ICS.
Fix it by assigning "setprop ro.ril.wake_lock_timeout 300", otherwise
wake_lock_timeout is set to default(60000) which is too long to act as
a timeout value.
See the CR for details.

Change-Id: Ib310937a327b502730d17c6c822fd8c866b4a81e
Signed-off-by: Shi Wenjun <b42754@freescale.com>
---
 arm2_6dq/init.rc     | 4 ++++
 arm2_6sl/init.rc     | 4 ++++
 evk_6sl/init.rc      | 4 ++++
 imx6/etc/init.rc     | 4 ++++
 sabreauto_6q/init.rc | 4 ++++
 sabresd_6dq/init.rc  | 4 ++++
 6 files changed, 24 insertions(+)

diff --git a/arm2_6dq/init.rc b/arm2_6dq/init.rc
index 3b92032..ec30d77 100755
--- a/arm2_6dq/init.rc
+++ b/arm2_6dq/init.rc
@@ -46,6 +46,10 @@ on boot
     # Set video overlay display mode
     setprop sys.VIDEO_OVERLAY_DISPLAY 1
 
+    # prepare for ril
+    setprop gsm.ril.delay 15
+    setprop ro.ril.wake_lock_timeout 300
+
     # Default backlight device
     setprop hw.backlight.dev "pwm-backlight.0"
     # Chmod/chown FSL specific sys entry
diff --git a/arm2_6sl/init.rc b/arm2_6sl/init.rc
index afc8eb1..97ac32f 100755
--- a/arm2_6sl/init.rc
+++ b/arm2_6sl/init.rc
@@ -42,6 +42,10 @@ on boot
     # Set video overlay display mode
     setprop sys.VIDEO_OVERLAY_DISPLAY 1
 
+    # Prepare for RIL
+    setprop gsm.ril.delay 15
+    setprop ro.ril.wake_lock_timeout 300
+
     # Default backlight device
     setprop hw.backlight.dev "pwm-backlight.0"
     # Chmod/chown FSL specific sys entry
diff --git a/evk_6sl/init.rc b/evk_6sl/init.rc
index 227001a..30aaddd 100644
--- a/evk_6sl/init.rc
+++ b/evk_6sl/init.rc
@@ -53,6 +53,10 @@ on boot
     # Indicate device type is tablet or SystemUI has problem without HW acceleration
     setprop sys.devicy.type tablet
 
+    # prepare for ril
+    setprop gsm.ril.delay 15
+    setprop ro.ril.wake_lock_timeout 300
+
 service wpa_supplicant /system/bin/wpa_supplicant \
     -Dnl80211 -e/data/misc/wifi/entropy.bin
     class late_start
diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index ceafc92..b09fcb6 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -384,6 +384,10 @@ on boot
     write /sys/class/android_usb/android0/iProduct $ro.product.model
     write /sys/class/android_usb/android0/iSerial $ro.serialno
 
+# Prepare for RIL
+    setprop gsm.ril.delay 15
+    setprop ro.ril.wake_lock_timeout 300
+
 # Set this property so surfaceflinger is not started by system_init
     setprop system_init.startsurfaceflinger 0
 
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 172a293..08c6e71 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -87,6 +87,10 @@ on boot
     chown system system /sys/devices/system/cpu/cpu3/online
     chmod 0660 /sys/devices/system/cpu/cpu3/online
 
+    # prepare for ril
+    setprop gsm.ril.delay 15
+    setprop ro.ril.wake_lock_timeout 300
+
     # Default backlight device
     setprop hw.backlight.dev "pwm-backlight.2"
     # Chmod/chown FSL specific sys entry
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index 2756423..1f6dedc 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -66,6 +66,10 @@ on boot
     # Set the density to 140dpi, default 128dpi is not good
     setprop ro.sf.lcd_density 140
 
+    # prepare for ril
+    setprop gsm.ril.delay 15
+    setprop ro.ril.wake_lock_timeout 300
+
 # change for FSL specific service
     chown system system /sys/devices/system/cpu/cpu0
     chown system system /sys/devices/system/cpu/cpu1
-- 
1.8.0

